sunpy/map/sources/hinode.py

```diff
@@ -1,8 +1,8 @@
 """Hinode XRT and SOT Map subclass definitions"""
-from sunpy.map import GenericMap
+import astropy.units as u
+from astropy.coordinates import Latitude, Longitude
 
-__author__ = ["Jack Ireland, Jose Ivan Campos-Rozo, David Perez-Suarez"]
-__email__ = "jack.ireland@nasa.gov"
```

Comment:
Should probably not remove this until we resolve https://github.com/sunpy/sunpy/issues/3650

Reply:
I think it should really go and we should decide on #3650.

Reply:
I agree, but given there's still disagreement lets resolve this in a different PR.

Reply:
If we don't decide tomorrow about this, I will undo these changes.